Submariner/SBDatabaseController.m

```diff
@@ -256,6 +256,13 @@ - (void)windowDidLoad {
                                                  name:@"SBSubsonicConnectionFailedNotification"
                                                object:nil];
 
+    // observe window state
+    [[NSNotificationCenter defaultCenter] addObserver:self
```

Comment:
The observer should be removed on `dealloc`

Reply:
Yeah, good catch